intl/msg_hash_es.h

```diff
@@ -3011,7 +3144,7 @@ MSG_HASH(
 	)
 MSG_HASH(
 	MENU_ENUM_LABEL_VALUE_VIDEO_SWAP_INTERVAL,
-	"Vertical Sync (Vsync) Swap Interval"
+	"Intervalo de intercambio de sincroniz. vertical (Vsync)"
```

Comment:
Keep labels short, sublabels can be as long as you want, but labels start to scroll left to right and looks awful
maybe a better one is
"Intervalo de intercambio de Vsync" 
(vsync it's under brackets in the upper term.)

Reply:
Ok. Noted.